src/Workspaces/CSharp/Portable/Extensions/SyntaxTokenExtensions.cs

```diff
@@ -71,41 +71,7 @@ public static bool IsFirstTokenOnLine(this SyntaxToken token, SourceText text)
         }
 
         public static bool SpansPreprocessorDirective(this IEnumerable<SyntaxToken> tokens)
-        {
```

Comment:
this moved to AbstractSyntaxFactsService so it could be shared between C# and VB instead of having it be duplicated.